tests/liveserver_fixtures.py

```diff
@@ -152,7 +153,23 @@ def post_auth():
 
     @app.route('/api/v1/<pid>/<lid>/predict', methods=["POST"])
     def predict_sinc(pid, lid):
+        body = flask.request.data
+        body = body.decode('utf-8').splitlines()
+        rows = len(body) - 1
+
+        try:
+            # for files with row_id as first column
+            first_row = int(body[1].split(',')[0]) % 10
```

Comment:
before this thing was always called with 10 requests long batch and was answering with static reply.

To check splitting logic, I have set max_batch_size to pretty small value (100 bytes) in test_regression tests, and now this code is called with 10 and 5 records batches, so I had to modify this endpoint to properly return top or bottom half of static output for 5 rows requests.